manifests/g/GetComposer/Composer/2.8.4/GetComposer.Composer.installer.yaml

```
@@ -0,0 +1,30 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GetComposer.Composer
+PackageVersion: 2.8.4
+InstallerLocale: en-US
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{7315AF68-E777-496A-A6A2-4763A98ED35A}_is1'
+ReleaseDate: 2024-12-11
+AppsAndFeaturesEntries:
+- DisplayName: Composer - PHP Dependency Manager
+  Publisher: getcomposer.org
+  ProductCode: '{7315AF68-E777-496A-A6A2-4763A98ED35A}_is1'
```

```suggestion
  DisplayVersion: 2.8.4
```

Does this modification solve the problem?

---

This doesn't work ...

<details><summary>Diff contents</summary>

```text
16:30:24 D:\...\winget-pkgs  [issues/56226 ≡ +0 ~1 -0 !] 0ms pwsh$ git diff                                                                                                            
diff --git a/manifests/g/GetComposer/Composer/2.8.4/GetComposer.Composer.installer.yaml b/manifests/g/GetComposer/Composer/2.8.4/GetComposer.Composer.installer.yaml
index 4f05a2b376e..3cfeac8749a 100644
--- a/manifests/g/GetComposer/Composer/2.8.4/GetComposer.Composer.installer.yaml
+++ b/manifests/g/GetComposer/Composer/2.8.4/GetComposer.Composer.installer.yaml
@@ -14,6 +14,7 @@ ProductCode: '{7315AF68-E777-496A-A6A2-4763A98ED35A}_is1'
 ReleaseDate: 2024-12-11
 AppsAndFeaturesEntries:
 - DisplayName: Composer - PHP Dependency Manager
+  DisplayVersion: 2.8.4
   Publisher: getcomposer.org
   ProductCode: '{7315AF68-E777-496A-A6A2-4763A98ED35A}_is1'
 ElevationRequirement: elevatesSelf
```

</details>
<details><summary>Installation logs</summary>

```text
--> Installing WinGet
--> Disabling safety warning when running installers
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

--> Configuring Winget
已启用管理员设置 'LocalManifestFiles'。
已启用管理员设置 'LocalArchiveMalwareScanOverride'。

--> Installing the Manifest 2.8.4

已找到 Composer - PHP Dependency Manager [GetComposer.Composer] 版本 2.8.4
此应用程序由其所有者授权给你。
Microsoft 对第三方程序包概不负责，也不向第三方程序包授予任何许可证。
此包需要以下依赖项：
  - 程序包
      PHP.PHP.8.4
(1/2) 已找到 Microsoft Visual C++ 2015-2022 Redistributable (x64) [Microsoft.VCRedist.2015+.x64] 版本 14.42.34433.0
此应用程序由其所有者授权给你。
Microsoft 对第三方程序包概不负责，也不向第三方程序包授予任何许可证。
正在下载 https://download.visualstudio.microsoft.com/download/pr/c7dac50a-e3e8-40f6-bbb2-9cc4e3dfcabe/1821577409C35B2B9505AC833E246376CC68A8262972100444010B57226F0940/VC_redist.x64.exe
  ██████████████████████████████  24.4 MB / 24.4 MB
已成功验证安装程序哈希
正在启动程序包安装...
已成功安装

(2/2) 已找到 PHP 8.4 [PHP.PHP.8.4] 版本 8.4.2
此应用程序由其所有者授权给你。
Microsoft 对第三方程序包概不负责，也不向第三方程序包授予任何许可证。
正在下载 https://windows.php.net/downloads/releases/php-8.4.2-Win32-vs17-x64.zip
  ██████████████████████████████  32.3 MB / 32.3 MB
已成功验证安装程序哈希
正在提取存档...
已成功提取存档
正在启动程序包安装...
已修改路径环境变量；重启 shell 以使用新值。
添加了命令行别名： "php"
已成功安装

正在下载 https://getcomposer.org/Composer-Setup.exe
  ██████████████████████████████  1.72 MB / 1.72 MB
已成功验证安装程序哈希
正在启动程序包安装...
已成功安装

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName                                                        DisplayVersion Publisher             ProductCode
-----------                                                        -------------- ---------             -----------
Composer - PHP Dependency Manager                                                 getcomposer.org       {7315AF68-E777-496A...
Microsoft Visual C++ 2015-2022 Redistributable (x64) - 14.42.34433 14.42.34433.0  Microsoft Corporation {804e7d66-ccc2-4c12...
PHP 8.4                                                            8.4.2          PHP Group             PHP.PHP.8.4_Microso...


PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> winget upgrade --accept-source-agreements -m .\manifests\g\GetComposer\Composer\2.8.4\
“msstore”源要求在使用前查看以下协议。
Terms of Transaction: https://aka.ms/microsoft-store-terms-of-transaction
源要求将当前计算机的 2 个字母的地理区域发送到后端服务才能正常工作，(例如"US")。

已找到 Composer - PHP Dependency Manager [GetComposer.Composer] 版本 2.8.4
此应用程序由其所有者授权给你。
Microsoft 对第三方程序包概不负责，也不向第三方程序包授予任何许可证。
此包需要以下依赖项：
  - 程序包
      PHP.PHP.8.4
(1/1) 已找到 PHP 8.4 [PHP.PHP.8.4] 版本 8.4.2
此应用程序由其所有者授权给你。
Microsoft 对第三方程序包概不负责，也不向第三方程序包授予任何许可证。
已成功验证安装程序哈希
正在提取存档...
已成功提取存档
```

</details>

---

Well... Then could you please mark this conversation as resolved for me... Thanks.
(I'm not a project maintainer or PR author, I don't know how I should mark the conversation as resolved)